pkg/placement/placement.go

```diff
@@ -215,6 +216,10 @@ func (p *Service) ReportDaprStatus(stream placementv1pb.Placement_ReportDaprStat
 				continue
 			}
 
+			for _, entity := range req.Entities {
+				monitoring.RecordActorHeartbeat(req.Id, entity, req.Name, req.Pod, p.clock.Now())
```

Comment:
How would this work in standalone mode?

Reply:
In self-hosted mode the pod name should just be empty, but the metric should still be tracked.

Reply:
Tested this in self-hosted mode and the metric reported like this in `placement-service:9090` metrics API
```
dapr_placement_actor_heartbeat_timestamp{actor_type="OrderActor",app_id="actor-service",host_name="10.1.2.10:54656",pod_name=""} 1.687829089e+09
```

Reply:
OK